cfme/tests/control/test_bugs.py

```diff
@@ -440,3 +449,48 @@ def _check_query():
             num_sec=600,
             message="Waiting for alert {} to appear in DB".format(alert.description)
         )
+
+
+@pytest.mark.parametrize(
+    "condition_class", CONDITIONS, ids=lambda condition_class: condition_class.__name__
+)
+def test_accordion_after_condition_creation(appliance, condition_class):
+    """
+    Bugzillas:
+        * 1683697
+
+    Polarion:
+        assignee: jdupuy
+        casecomponent: Control
+        caseimportance: medium
+        initialEstimate: 1/12hr
+
+    For this test, we must make a condition 'manually' and so that we can access the view
+    during the condition creation.
+    """
+    if (
+        BZ(1683697, forced_streams=["5.9", "5.10"]).blocks and
+        (condition_class in BAD_CONDITIONS or appliance.version < "5.10")
+    ):
+        pytest.skip("Skipping because {} conditions are impacted by BZ 1683697".format(
+            condition_class.__name__
+        ))
+    condition = condition_class(
+        appliance.collections.conditions,
+        fauxfactory.gen_alpha(),
+        expression="fill_field({} : Name, IS NOT EMPTY)".format(
+            condition_class.FIELD_VALUE)
+    )
+    view = navigate_to(condition, "Add", force=True)
+    # first fill description, expression, and notes
+    view.fill({
```

Comment:
Required : Please use create method instead

Reply:
@sshveta most recent push has been updated to use the create method.